CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,32 @@
+# `Phoebus` Community Code of Conduct
```

Comment:
We should not add a code of conduct to phoebus. It's a LANL code, and we already have a code of conduct at LANL that we cannot override. If external participants become a problem relative to LANL standards we can limit their participation appropriately.

Reply:
If this is a concern, we can instead link to the LANL code of conduct. https://cdn.lanl.gov/files/code-of-conduct-1_312ce.pdf

We could summarize the most relevant part of the code of that document which is on page 36:
```
- Foster a mutually respectful working environment that is free from discrimination,
intimidation, and harassment (including sexual harassment).
- Do not discriminate against others on the basis of any characteristic protected by law
or LANL policy.
- Do not engage in behaviors that create an offensive, hostile, or intimidating work
environment, including shouting, abusive language, threats of violence, the use of
obscenities or other non-verbal expressions of aggression, horseplay such as kicking
co-workers’ chairs or using personal insults, or the use of offensive nicknames or other
derogatory remarks.
- Avoid behavior that a reasonable person would find to be demeaning or humiliating.
```

Reply:
Alternatively, we could link to [this code of conduct](https://github.com/karlgroves/dontbeadick), which I am a fan. (This is mostly a joke. It doesn't address @brryan 's concern.)

Reply:
I'm open to pointing to LANL code of conduct if this is a concern.

Reply:
No concerns from me with replacing this PR's COC with references to the LANL code of conduct instead. @Yurlungur's idea of a relevant summary is a good one.